README.md

```diff
@@ -1,2 +1,21 @@
 # zap-cli
-A Node.js CLI command for searching for, or opening a YouTube video with Zen Audio Player.
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zen-audio-player/zen-audio-player.github.io?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Build Status](https://travis-ci.org/zen-audio-player/zen-audio-player.github.io.svg?branch=master)](https://travis-ci.org/zen-audio-player/zen-audio-player.github.io)
+[![GitHub Stats](https://img.shields.io/badge/github-stats-ff5500.svg)](http://githubstats.com/zen-audio-player/zen-audio-player.github.io)
```

Comment:
I added the correct zap-cli links but still not sure about the glitter link because it was brining me to a 404. That said, I am now going to look into #1 now and work to begin setting them up.